### DISCUSSION TITLE
Save JQL queries for future use

### DISCUSSION THREAD

**Discussion by drewhousman1010** [ParabolInc/parabol#6111] [Ideas]
A [user ](https://app.hubspot.com/contacts/3888472/contact/64282151) 🔒 writes:

> We use a JQL query to pull in JIRA items for estimation. Is there a way to save the query that is used? I have to save it off myself and copy/paste it in every two weeks.

**Comment by tianrunhe** [comment]
If they **have added** any issues from the result of a JQL query, that query will be automatically saved. It's just a little bit hard to find: It's under 🔍.

<img width="656" alt="Screen Shot 2022-02-23 at 10 25 13 AM" src="https://user-images.githubusercontent.com/1879975/155383488-93de95d1-ac83-4143-919e-ad9db672e3fd.png">
